_docs/kubernetes/provision-volumes.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: guide
+title: StorageOS Docs - Provision volumes
+anchor: kubernetes
+module: kubernetes/provision-volumes
+---
+
+This document describes how to provision StorageOS volumes. Administrators
+define storage profiles using Kubernetes StorageClasses, and users request
+storage volumes using PersistentVolumeClaims (PVCs).
+
+Create all examples below:
+```bash
+git clone https://github.com/storageos/deploy.git storageos
+kubectl create -f storageos/examples/
+```
+
+## Storage class
```

Comment:
I would move the actions done by the cluster administrator (create StorageClass, secret etc) into a separate doc.  I'm guessing that many developers will go straight to the "provision volumes" page and only help to create/verify/delete PVC will be relevant to them.

Reply:
Done.